src/sentry/event_manager.py

```diff
@@ -601,10 +601,17 @@ def save(self, project_id, raw=False, assume_normalized=False, start_time=None,
         # Load attachments first, but persist them at the very last after
         # posting to eventstream to make sure all counters and eventstream are
         # incremented for sure.
-        attachments = get_attachments(cache_key, job["event"])
-        for attachment in attachments:
-            key = "bytes.stored.%s" % (attachment.type,)
-            job["event_metrics"][key] = (job["event_metrics"].get(key) or 0) + len(attachment.data)
+        attachments = []
+        for attachment in get_attachments(cache_key, job["event"]):
+            try:
+                key = "bytes.stored.%s" % (attachment.type,)
```

Comment:
_nit_: Only do `attachment_data = attachment.data` in here like in the consumer, and the rest in `else:`?

Reply:
done